tests/test_spd_matrices.py

```diff
@@ -250,7 +259,7 @@ def test_log_euclidean_inner_product(self):
 
         self.assertAllClose(result, expected)
 
-    @geomstats.tests.np_and_tf_only
+    @geomstats.tests.np_only
```

Comment:
Let's not break tests in other backends, what was the issue?

Reply:
Theses tests yied 'shape mismatch' I can't figure out...
I'm really surprised as these functions shouldn't be affected by the commit